templates/definition/meter/solaredge-hybrid.yaml

```diff
@@ -9,6 +9,9 @@ requirements:
     de: |
       Nur ein System kann und darf zeitgleich eine Modbus TCP-Verbindung zum Wechselrichter haben!
       Für die optionale Batteriesteuerung muss StorageConf_CtrlMode (0xE004) auf 4 "Remote" stehen.
+      Funktion der Batteriesteuerung ist nur bei aktiviertem Batteriemodus "Nutzungszeit" möglich.
+      Anscheinend wird der StorageConf_CtrlMode (0xE004) bei deaktivierung des Modus von "Nutzungszeit" 
```

Comment:
Braucht es den zweiten Satz? Oder komme ich aus dem Zustand auch wieder raus, indem ich in "Nutzungszeit" gehe? Dann würde der erste Satz ja reichen.

Reply:
Ja, es reicht wieder auf Nutzungszeit zu stellen.